contribution_stats.py

```
@@ -0,0 +1,74 @@
+
+CONFIG_FILE_EXTENSIONS = (".json", ".yml", ".yaml", ".ini", ".conf", ".toml")
+
+
+def is_text_file(filepath):
+    # Check for binary file by scanning for null bytes.
+    try:
+        with open(filepath, "rb") as f:
+            chunk = f.read(4096)
+        if b"\0" in chunk:
+            return False
+        return True
```

Ruff (0.12.2)

Return the condition `b"\x00" not in chunk` directly

Replace with `return b"\x00" not in chunk`

(SIM103)